apps/web/src/components/features/properties/PropertyDetail.tsx

```
@@ -0,0 +1,298 @@
+            <div className="space-y-2 mb-6">
+              <label htmlFor="guest-count" className="text-sm font-medium">Guests</label>
+              <div className="flex items-center border rounded-md p-2 bg-background">
+                <Users className="h-5 w-5 text-muted-foreground mr-2" />
+                <select 
+                  id="guest-count"
+                  className="border-0 p-0 focus:outline-none w-full bg-transparent"
+                  value={bookingData.guests}
+                  onChange={(e) => setBookingData({ ...bookingData, guests: Number(e.target.value) })}
+                >
+                  {[...Array(property.maxGuests)].map((_, i) => (
+                    <option key={i + 1} value={i + 1}>
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)